setuptools/tests/test_sdist.py

```diff
@@ -122,9 +122,9 @@ def touch(path):
 def symlink_or_skip_test(src, dst):
     try:
         os.symlink(src, dst)
-        return dst
     except (OSError, NotImplementedError):
         pytest.skip("symlink not supported in OS")
+    return dst
```

Comment:
Ouch. This is probably wrong! In the absence of a return statement, this used to return `None` in case of `OSError`/`NotImplementedError`.

Reply:
Those were manual fixes as far as I can remember, not the result of `--fix`. I should have written that...